cd/scripts/release-controller.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+USAGE="
+Usage:
+  $(basename "$0")
+
+Publishes the controller image and helm chart for an ACK service controller as part of prowjob
+when a service controller repository is tagged with a semver format '^v\d+\.\d+\.\d+$'
+See: https://github.com/aws-controllers-k8s/test-infra/prow/jobs/jobs.yaml for prowjob configuration.
+
+Environment variables:
+  REPO_NAME:                Name of the service controller repository. Ex: apigatewayv2-controller
+                            This variable is injected into the pod by Prow.
+  PULL_BASE_REF:            The value of tag on service controller repository that triggered the
+                            postsubmit prowjob. The value will be in the format '^v\d+\.\d+\.\d+$'.
```

Comment:
According to the below, the value might also be "stable"?

Reply:
Yes, Thanks! Updated.